docs/bbs/index.md

```diff
@@ -16,3 +15,7 @@
 * Other files
     * [font.arc, fonten.arc](font.md)
 
+## Dictionaries
+
+* [Worlds](dictionary/worlds.md)
```

Comment:
Is moving worlds.md to the dictionary dir coming in an other commit?

Reply:
I actually forgot... Thank you.